tests/test_Sim.py

```diff
@@ -210,7 +210,7 @@ def check_data(self, how_many, tdc_en=False, start=0):
         if tdc_en:
             exp_tdc = np.array([135] * how_many, dtype=np.uint32)
             ret_tdc = ret[~tlu_word] >> 20
-            self.assertEqual((ret_tdc / 2).tolist(), (exp_tdc / 2).tolist())
+            self.assertEqual((ret_tdc // 2).tolist(), (exp_tdc // 2).tolist())

```

Comment:
Why not:
```python
self.assertEqual(ret_tdc // 2, exp_tdc // 2)
```

It does the same though ;-)

Reply:
This gives ``ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()``. Apparently ``self.assertEqual`` works only with lists. One could use
``` python
np.testing.assert_array_equal(ret_tdc // 2, exp_tdc // 2)
```
in order to check directly two arrays..